election_2020/model_prediction.py

```diff
@@ -0,0 +1,85 @@
+import os
+import pandas as pd
+import json
+
+from app.BigQuery_service import BigQueryService
+from app.model_storage import ModelStorage
+from app import DATA_DIRPATH, ELECT_FIRPATH
+
+bq = BigQueryService()
+
+print("LOAD EMBEDDINGS:")
+#filename = 'embedding_user_sample_testing200.csv' #small sample file for test the code
+#filename = 'embedding_openai_user_sample_max_50_size_10000.csv'
+#filepath = os.path.join(ELECT_FIRPATH, filename)
+##print(filepath)
+#if os.path.exists(filepath):
+#    df = pd.read_csv(filepath)
+#    print(df.shape) #make sure we have read the file
+#else:
+#    print(f"File not found: {filepath}")
+sql = '''SELECT * 
```

Comment:
Let's update this query to fetch user id and embeddings from the embeddings table, left join to the preds table, where user id in preds table is null.